plasma_framework/contracts/mocks/exits/utils/ExitBountyWrapper.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity 0.5.11;
+
+import "../../../src/exits/utils/ExitBounty.sol";
+
+contract ExitBountyWrapper {
+
+    function processStandardExitBountySize() public view returns (uint256) {
```

Comment:
I think we need this function in the exit game contract directly. Users would need to know what bounty they should pass into. (Same as what we did for bond)

Reply:
👍 will take the gas price as an arg and add this func to exit game